File: TesseractApp/app/src/main/java/com/tesseract/light/Light.kt
package com.tesseract.light

data class Light(var name: String, val description: String, var colors: ArrayList<String>, val colors_parameters: ArrayList<String>, var intensity: Int?, val pattern: String?, var speed: Float?, var modifier: String?, var face: String?) {

    override fun toString(): String {
        return name
    }
}

File: Tesseract/Light/Light.py

class Light:
	name = ...  # type: str
	colors = ...  # type: list
	description = ...  # type: str
	colors_parameters = ...  # type: list

	def __init__(self, name, description, colors, colors_parameters):
		self.name = name
		self.description = description
		self.colors = colors
		self.colors_parameters = colors_parameters
